src/hooks/useComment.ts

```diff
@@ -0,0 +1,54 @@
+import { ChangeEvent, FormEvent, useState } from 'react';
+import { useParams } from 'react-router-dom';
+
+import { deleteStoryComment, postStoryComment } from '../apis/story';
+import { ERROR_MESSAGES } from '../constants/errorMessages';
+
+export const useCommentForm = () => {
+  const [comment, setComment] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+  const { storyId } = useParams();
+
+  const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
+    setComment(e.target.value);
+  };
+
+  const handleSubmit = async (e: FormEvent) => {
+    e.preventDefault();
+    setIsLoading(true);
+
+    if (comment.trim().length === 0) {
+      alert('댓글 내용을 입력해 주세요.');
+    } else {
+      try {
+        if (!storyId) throw Error;
```

Comment:
혹시 이러면 구체적으로 어떤 에러인지 콘솔창에서 확인할 수 있나요? 이건 정말 제가 잘 몰라서요. 이렇게 해도 어떤 Error인지 콘솔창에 말해주는지 궁금하네요. 사소한 거니까 넘어갑시다!

```suggestion
if (!storyId) throw Error('잘못된 스토리 조회입니다.(storyId is not available)');
```

Reply:
@metacode22 오 저런 급하게 쳐내느라 제대로 처리를 못했네요..!

그냥 throw Error로 하면 아래와 같이 아주 불친절한 에러 로그가 뜹니당
![스크린샷 2023-01-12 오후 7 00 03](https://user-images.githubusercontent.com/63575891/212037103-e9ead8f3-3023-4fa9-8ec4-d65d6e1ed1fc.png)

제안해주신 대로 했는데 훨씬 나아졌어요!! 감사합니다~
![스크린샷 2023-01-12 오후 7 00 37](https://user-images.githubusercontent.com/63575891/212037243-b65d9d83-7090-445d-80d2-41b6f9276fcd.png)